### PR TITLE
updated to post a job under job/internship

### PR DESCRIPTION
**Pull request by jaatheone**
new functionality under job/internship

### CODE REVIEW

File: src/main_menu.py [hunk 1]
```diff
@@ -1,18 +1,90 @@
...
+            job_description = input("Job Description: ")
+            employer = input("Employer: ")
+            location = input("Location: ")
+            salary = input("Salary: ")
+            username = get_user()
+            username2 = username[1]
+            job_entry(username,job_title,job_description,employer,location,salary)
+        elif selection == 2:
+            return
+        elif selection != "1" or selection != "2":
```
Comment (GrayThinker) [inline src/main_menu.py:72]: Is there a reason we are comparing strings here?

**Review by cread99 — APPROVED** [review]

File: src/main_menu.py (hunk 1 above)
Comment (proyectoramirez) [reply src/main_menu.py:72]:
```suggestion
else:
```

**Review by proyectoramirez — CHANGES_REQUESTED** [review]

**Review by kbsanchez — APPROVED** [review]

**Comment by kbsanchez** [comment]
What do you guys want to do with this pull request? @proyectoramirez @jaatheone it has two approvals but the changes are still requested. 

Since its behind main I think the best approach may be to make an edit to the main_menu.py file and include it in your upload when you/you guys create a pull request for testing. If there's a better approach go for it

File: src/main_menu.py [hunk 2]
```diff
@@ -1,18 +1,90 @@
...
+            job_description = input("Job Description: ")
+            employer = input("Employer: ")
+            location = input("Location: ")
+            salary = input("Salary: ")
+            username = get_user()
+            username2 = username[1]
+            job_entry(username,job_title,job_description,employer,location,salary)
+        elif selection == 2:
+            return
+else:
```
Comment (proyectoramirez) [inline src/main_menu.py:72]:
```suggestion
        else:
```

**Review by proyectoramirez — APPROVED** [review]

**Pull request merged**